org_fedora_oscap/data_fetch.py

```diff
@@ -90,8 +96,7 @@ def fetch_data(url, out_file, ca_certs=None):
     out_dir = os.path.dirname(out_file)
     utils.ensure_dir_exists(out_dir)
 
-    if url.startswith("http://") or url.startswith("https://") \
-       or url.startswith("ftp://"):
+    if can_fetch_from(url):
         _fetch_http_ftp_data(url, out_file, ca_certs)
```

Comment:
Probably a time for a better name for this function. Could you please update the pull request with such a change? Maybe something like ``_curl_fetch()``?